plugins/woocommerce-blocks/assets/js/previews/shipping-rates.ts

```diff
@@ -12,6 +12,7 @@ import { API_SITE_CURRENCY, displayForMinorUnit } from './utils';
 
 // Get local pickup locations from the settings and format into some preview shipping rates for the response.
 const localPickupEnabled = getSetting< boolean >( 'localPickupEnabled', false );
+const localPickupTitle = getSetting< string >( 'localPickupText', __( 'Local pickup', 'woocommerce' ) );
```

Comment:
Apologies, this should be:
```suggestion
const localPickupTitle = getSetting< string >(
	'localPickupText',
	__( 'Local pickup', 'woocommerce' )
);
```

Reply:
Resolved linting issue.